pnc-mock/src/main/java/org/jboss/pnc/mock/coordinator/BuildCoordinatorMock.java

```diff
@@ -72,17 +73,20 @@ public BuildTask build(BuildConfiguration buildConfiguration, User user, boolean
         return Mockito.mock(BuildTask.class);
     }
 
-    @Override
     public BuildSetTask build(BuildConfigurationSet buildConfigurationSet, User user, boolean rebuildAll) throws CoreException {
         logger.warn("Invoking unimplemented method build");
         return Mockito.mock(BuildSetTask.class);
     }
 
-    @Override
     public List<BuildTask> getSubmittedBuildTasks() {
         return activeTasks;
     }
 
+    @Override
```

Comment:
Yes the override should stay at other methods as well. I don't know why I removed it.